### DISCUSSION TITLE
Willkommen

### DISCUSSION THREAD

**Discussion by SsaraBab** [SsaraBab/Gruppenarbeit-WA#37] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MawKi2** [comment]
Hier sind die Ergebnisse aus der Anwendung von a und b:
Die deskriptive Analyse der metrischen Variablen ergab ein Durchschnittsalter von 25,3 Jahren (im Median 25) der Befragten, einen Mittelwert von 4,3 beim Interesse in Mathe und einen Mittelwert von 4 beim Interesse in Programmierung. Die Standardabweichung beträgt in allen Fällen ca. 2,1.
Bei den kategorialen zeigt sich eine Häufung bei den 23-27 jährigen (insg. 0,8 rel. Häufigkeit) beim Alter. Beim Mathe Interesse gibt es ein große Häufigkeit der 7, alle anderen Wertesind sehr gleich verteilt. Beim Interesse Programmierung sind 1, 2 und 7 häufiger vertreten. Nach Fächern ist Mathe im Vergleich wesentlich schwächer vertreten (nur 0,07 rel. Häufigkeit). Die Aufteilung nach Mathe LK ist nahezu 50/50.

**Comment by reneesch** [comment]
Danke, Marc! 

@ibrahim-tudo und @Moshirimozafar, mögt ihr vielleicht was zu c) bis f) und der freiwilligen Funktion sagen? Man kann hier zu Kommentaren auch Dateien, z. B. Grafiken, hinzufügen.

**Reply by reneesch** [reply]
Hm, oder sollen wir die Diskussion vielleicht in Saras Post "Ergebnisse" verlegen? Ist übersichtlicher. @MawKi2 kannst du es dort nochmal posten vielleicht
?

**Reply by Moshirimozafar** [reply]
Ich glaube, dass wir die Diskussion in Ergebnisse verlegen sollten... ;)

**Reply by MawKi2** [reply]
habe es nochmal in Saras gepostet
